internal/pkg/skuba/addons/cilium.go

```diff
@@ -273,13 +273,15 @@ spec:
           initialDelaySeconds: 120
           failureThreshold: 10
           periodSeconds: 10
+	  timeoutSeconds: 5
```

Comment:
Incorrect indentation. `timeoutSeconds` should have exactly the same number of spaces before as `periodSeconds`. You probably have a tab character here. Please use only spaces.

Reply:
ditto

Reply:
Oops, vim using tabs. Sorry.

Reply:
https://github.com/cilium/cilium/blob/7f617ed02923aa95569eb0728a840a0bdeeb62d8/install/kubernetes/cilium/charts/agent/templates/daemonset.yaml
Also since we are touching the 'timeoutSeconds' in this file, it would be good to fix the other default values as well.
'periodSeconds' is set to '30' in upstream
'failureThreshold' is set to '3'